src/models/filter.rs

```diff
@@ -333,6 +355,17 @@ impl SourceCodeUnit {
       }
     }
 
+    // If an outermost enclosing node is provided
+    let query = instantiated_filter.outermost_enclosing_node();
+    if !query.get_query().is_empty() {
+      debug!("here");
```

Comment:
artifact ?

Reply:
had to remove the import too, the linter failed :| haha